.github/ISSUE_TEMPLATE/---releng-checklist.md

```diff
@@ -0,0 +1,53 @@
+---
+name: "✅ Releng checklist"
+about: Keep track of the release activities
+title: 'Releng checklist - vX.X'
+
+---
+
+##### Sprint Release Owner: _[name here]_
+
+#### Tuesday #1
+
+- [ ] Upgrade a-c
+
+#### Wednesday #1
+
+- [ ] Export strings (*if applicable*) ([instructions](https://github.com/mozilla-mobile/firefox-tv/wiki/Localization#exporting-strings-for-translation))
+- [ ] Create GitHub pre-release build ([instructions](https://github.com/mozilla-mobile/firefox-tv/wiki/Releng-Checklist#final-builds))
+- [ ] Release LAT ([instructions](https://developer.amazon.com/docs/app-testing/live-app-testing-getting-started.html))
+
+#### Wednesday #2 (soft code freeze)
+
+- [ ] Create PR for bumping version and updating changelog
```

Comment:
Do we bump to the LAT version (i.e. LAT2?)? Or do we use the release version? We should specify.